common/export.py

```diff
@@ -349,11 +309,16 @@ def plot_impacts(process_name, impacts_smp, impacts_bw, folder, impacts_py):
         for t in impacts_py.keys()
         if t in impacts_smp.keys() and t in impacts_bw.keys()
     ]
-    nf = compute_normalization_factors(impacts_py)
+    factors = get_normalization_weighting_factors(impacts_py)
 
-    # Calculate aggregated values for comparison
-    simapro_values = [calculate_aggregate(impacts_smp, nf["ecs"])]
-    brightway_values = [calculate_aggregate(impacts_bw, nf["ecs"])]
+    simapro_values = [
+        impacts_smp[trigram] / factors["pef_normalizations"][trigram]
+        for trigram in trigrams
+    ]
+    brightway_values = [
```

Comment:
As impact values are normalized but not weighted could we change the ylabel from `Impact Values` to `Impacts Values (normalized, non weighted)`